### PR TITLE
Update get-all-recommendations.md

### PR DESCRIPTION
**Pull request by msbemba**
Update Odata filter query

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5921

File: microsoft-365/security/defender-endpoint/get-all-recommendations.md [hunk 1]
```diff
@@ -37,6 +37,19 @@ ms.custom: api
...
 
+
+## API description
+
+Returns information about all security recommendations affecting the organization.
+
+*URL:** GET: /api/recommendations
+<br>Supports [OData V4 queries](https://www.odata.org/documentation/).
+<br>OData supported operators:
+<br>```$filter``` on:  ```id```, ```productName```, ```vendor```, ```recommendedVersion```, ```recommendationCategory```, ```subCategory```, ```severityScore```, ```remediationType```, ```recommendedProgram```, ```recommendedVendor``` and ```status``` properties.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/get-all-recommendations.md]:
```suggestion
<br>```$filter``` on:  ```id```, ```productName```, ```vendor```, ```recommendedVersion```, ```recommendationCategory```, ```subCategory```, ```severityScore```, ```remediationType```, ```recommendedProgram```, ```recommendedVendor```, and ```status``` properties.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@DulceMontemayor Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@DulceMontemayor Gentle reminder

**Comment by yogkumgit** [comment]
@tiburd

File: microsoft-365/security/defender-endpoint/get-all-recommendations.md [hunk 2]
```diff
@@ -37,6 +37,19 @@ ms.custom: api
 
 Retrieves a list of all security recommendations affecting the organization.
 
+
+## API description
+
+Returns information about all security recommendations affecting the organization.
+
+*URL:** GET: /api/recommendations
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/get-all-recommendations.md]:
```suggestion
*URL:* GET:/api/recommendations
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba - apologies Sri, I have missed this one. Please look at the Rich View and see if you agree. Thanks.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**